website/content/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -88,6 +88,57 @@ Registering the external services with Consul is a multi-step process:
 
 ### Register external services with Consul
 
+There are two ways to register an external service with Consul:
+1. If you have [`TransparentProxy`](/docs/k8s/helm#v-connectinject-transparentproxy) enabled, you can declare external endpoints with the [Destination](/docs/connect/config-entries/service-defaults#terminating-gateway-destination) field of `service-defaults`.
+1. You can add the service as a Node in the Consul catalog.
```

Comment:
```suggestion
1. You can add the service as a node in the Consul catalog.
```
If we mean Kubernetes Node, then we should capitalize to distinguish between nodes in Consul and Nodes in Kubernetes.

Reply:
Ah, TIL. 🙏